blender/arm/write_data.py

```diff
@@ -569,12 +570,19 @@ def write_compiledglsl(defs, make_variants):
             f.write('#define GBUF_IDX_1 1\n')
 
             idx_emission = 2
+            idx_refraction = 2
             if '_gbuffer2' in wrd.world_defs:
                 f.write('#define GBUF_IDX_2 2\n')
                 idx_emission += 1
+                idx_emission += 1
```

Comment:
Should this be `idx_refraction`?

Reply:
To be frank, I did not manage any kind of mixing with basecolor when refraction is used.
It does refract, more or less, and that's all. No color is taken into account. This is because the refractive objects must not contribute to the render, but actually I can manage that easily.